### ISSUE TITLE
Unable to Create Session (Protobuf Parsing Error)

### ISSUE THREAD

**Issue by jdp8** [dakenf/stable-diffusion-webgpu-minimal#2]
I've been trying to run this project locally but I keep getting the following error: "Error: Can't create a session. ERROR_CODE: 7, ERROR_MESSAGE: Failed to load model because protobuf parsing failed."

I cleared the site data as mentioned in the FAQ but this did not fix the issue. Is there anything else that is needed to fix this issue? Any assistance would be appreciated. Thank you!

**Comment by dakenf** [comment]
Can you make a screenshot of Network tab in dev tools?

**Comment by jdp8** [comment]
Sure! This is what appears after pressing 'LOAD MODEL'.

<img width="1918" alt="Screenshot 2023-08-16 at 4 51 51 PM" src="https://github.com/dakenf/stable-diffusion-webgpu-minimal/assets/68886123/d645a3de-728b-4aa3-b25b-73e3eb2635cd">

**Comment by dakenf** [comment]
I think i know where the problem is. Please download these model files https://huggingface.co/aislamov/stable-diffusion-2-1-base-onnx/tree/9f697c96d42e5c09437ff14b0a2b287366ce488d and put them to `public\models\aislamov\stable-diffusion-2-1-base-onnx`

This happens because `create-react-app` returns HTML page with 200 code instead of 404. And huggingface hub utils from transformers.js treat it as a correct model response

I will update the code to handle it in next few days (and also release new version of onnx runtime that takes ~2.2sec for one step instead of a minute)

**Comment by jdp8** [comment]
I tried your solution but it did not work until I switched to a previous commit (the SHA is 13cf3691d387da2f5f473ccadb08d460dd389856). After this I was able to download the model files from the `public\models\aislamov\stable-diffusion-2-1-base-onnx` directory. However when I pressed 'RUN' the canvas remained black. In other words, an image was never generated. Upon checking the 'Application' tab in dev tools I found that the `tokenizer/merges.txt` and `tokenizer/vocab.json` files were not downloaded properly. Maybe this could be another issue?

<img width="1916" alt="Screenshot 2023-08-17 at 6 08 50 PM" src="https://github.com/dakenf/stable-diffusion-webgpu-minimal/assets/68886123/c13b94a1-c909-4dc8-bea3-612edfdee831">

Also I tried to run your deployed project but it threw the following error which I assume was due to the commit you did a couple of days ago to the Hugging Face repo (I saw that the model weight files were added):

<img width="1827" alt="Screenshot 2023-08-17 at 6 17 50 PM" src="https://github.com/dakenf/stable-diffusion-webgpu-minimal/assets/68886123/e7ff9300-4b34-42b5-974a-9fd018194af8">

I will wait for your changes and test the project then. Thank you for your assistance!
